### PR TITLE
austin.intro.1.andrew.flater

### PR DESCRIPTION
**Pull request by aflater**

### CODE REVIEW

File: 01Lesson/index1 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Your first website
```
Comment (ivome) [inline 01Lesson/index1:1]: Most files should have a file extension like `.html` or `.md`

File: 02Lesson/01Homework/index.html [hunk 2]
```diff
@@ -1,8 +1,32 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <body style="background-color: aqua"> </p>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:5]: No body element allowed in `head` section. Closing `p` tag before opening tag.

File: 02Lesson/01Homework/index.html [hunk 3]
```diff
@@ -1,8 +1,32 @@
...
+  <title>Andrews Webpage bio</title>
 </head>
 <body>
+<h1>ANDREW ROBERT FLATER</h1>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:9]: Indentation: All child elements of body element need to be indented by 2 spaces.

File: 02Lesson/01Homework/index.html [hunk 4]
```diff
@@ -1,8 +1,32 @@
...
+<H2>3210 esperanza xing, Austin TX 78758</h2>
+  <img src="https://lh3.googleusercontent.com/-cTgHmIB6UyQ/AAAAAAAAAAI/AAAAAAAAABI/s-iLuk7yTOg/s180-c-k-no/photo.jpg"/>
+<h2>BIO</h2>
+<p> Born in Portland Oregon, moved to Austin Texas in 2011. Got married in 2014. Currently live in north Austin.
+  Going to school to be a Computer Programmer.
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:14]: Missing closing `p` tag.

File: 02Lesson/01Homework/index.html [hunk 5]
```diff
@@ -1,8 +1,32 @@
...
+  <ol>
+      <li>Favorite Sports</li>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:16]: Missing closing `ol` tag.

File: 02Lesson/01Homework/index.html [hunk 6]
```diff
@@ -1,8 +1,32 @@
...
+  <ul>
+      <li>Soccer</li>
+      <li>American Football</li>
+      <li>Baseball</li>
+      <li>Basketball</li>
+  </ul>
+<h2>My favorite Websites</h2>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:23]: Indentation...

File: 02Lesson/01Homework/index.html [hunk 7]
```diff
@@ -1,8 +1,32 @@
...
+  <a href="https://www.chelseafc.com">Chelsea FC<img src="http://www.chelseafc.com/content/cfc/en/ref/clubs/en/chelsea.img.png"/>
```
Comment (ivome) [inline 02Lesson/01Homework/index.html:24]: Missing closing `a` tag.

File: 02Lesson/01Project/index.html [hunk 8]
```diff
@@ -1,9 +1,39 @@
...
+<ul>
+  <li style="color:red">Dog walking</li>
+  <li style="color:red">Overnight pet sitting</li>
+  <li style="color:red">Grooming</li>
+  <li>Pet Taxi Service</li>
+</ul>
+
+
+<p style =”font size: 40px; color: blue; text-align:center";font-family: Arial”><em>Voted best dog walker in Austin Texas 2002</em></p>
+<a href="https:///www.skylinepetcareaustin.com/"> CLICK HERE FOR CLIENT REVIEWS</a>
```
Comment (ivome) [inline 02Lesson/01Project/index.html:34]: Check the indentation for the page

File: 02Lesson/02Project/index.html [hunk 9]
```diff
@@ -1,10 +1,43 @@
...
+        </li>
+        <li>
+          <a href="https://www.kxan.com/">Local News</a>
+        </li>
+        <li>
+          <a href="https://www.http://www.wsj.com/news/economy/">Economy<a/>
+        </li>
+      </ul>
+    </nav>
+        </h1>
```
Comment (ivome) [inline 02Lesson/02Project/index.html:27]: Element `nav` not allowed as child element `h1`. Indentation of closing `h1` tag is wrong.

File: 02Lesson/02Project/index.html [hunk 10]
```diff
@@ -1,10 +1,43 @@
...
+    <main>
+    <section>
+    <article>
+          <a href="https://www.nbcnews.com">Election Day 2016</a>
```
Comment (ivome) [inline 02Lesson/02Project/index.html:31]: Too many spaces as indentation.

File: 02Lesson/02Project/index.html [hunk 11]
```diff
@@ -1,10 +1,43 @@
...
+          <img src="http://i2.cdn.turner.com/cnnnext/dam/assets/161009220353-trump-clinton-2nd-debate-02-exlarge-169.jpg">
+    </article>
+    <section>
+    <aside>
```
Comment (ivome) [inline 02Lesson/02Project/index.html:35]: Indentation.

**Review by ivome — CHANGES_REQUESTED** [review]
You have a few issues in your code that are really easy to fix:
I would go through your code again and first try to fix the indentation. It might be easier to just create the documents from scratch and check often. Maybe have someone in a learning group look over your code while you do this. 
Then save often and check for the W3C validation errors in Atom. Once you have done that, all the errors in your code should be fixed... ;-) 
Without indentation you will keep making a lot of unnecessary mistakes. 
Keep it up!!

**Comment by ivome** [comment]
How you can check your result: See if the tests in your pull request are passing. You should have a green checkmark next to your pull request. Click on the details for circleci if you see any errors, that will tell you where to look in your code.

**Comment by aflater** [comment]
Thank you Ivo. I'm meeting with the group today to go over stuff. See you
monday!

**Comment by aflater** [comment]
Hey Ivo, 
it's Andrew flater just want to see when you think the grades will be up?

Andrew

**Comment by ivome** [comment]
Great Job catching up!! 
You build a really nice looking final project. Unfortunately there is a few things missing like a map and forms and one of those optional javascript modules, or did I miss something? Also, did you mean to place the content boxes at the bottom next to each other? 
The task for lesson 12 project was to replicate the given layout. I would recommend to build that, since it requires some interesting applications of the grid system.

**Comment by aflater** [comment]
Hi Ivo,

 thank you so much for the feedback. For practice I'm just gonna start over and try to redo everything also bought several books to get me ready for next semester. I've already started looking at it. Thanks again for a great semester! 
Hi 
Andrew

**Pull request closed without merging**